docker-compose.yml

```diff
@@ -72,7 +79,53 @@ services:
       - isard-database
       - isard-hypervisor
 
+  isard-grafana:
+    environment:
+      - ISARD_PUBLIC_DOMAIN=${ISARD_PUBLIC_DOMAIN}
+    container_name: isard-grafana
+    volumes:
+      - "/opt/isard/grafana/grafana/data:/grafana/data"
+      - "/opt/isard/grafana/graphite/storage:/opt/graphite/storage"
+      - "/opt/isard/grafana/graphite/conf:/opt/graphite/conf"
+    ports:
+      - target: 3000
+        published: 3000
+        protocol: tcp
+        mode: host
+    networks:
+      - isard_network
+    image: isard/grafana:${TAG:-latest}
+    build:
+      context: .
+      dockerfile: dockers/grafana/Dockerfile
+    restart: unless-stopped
+
+  isard-squid:
+    environment:
+      - SPICE_HOSTS=${SPICE_HOSTS}
+      - SPICE_PROXY_PORT=${SPICE_PROXY_PORT}
+    container_name: isard-squid
+    ports:
+      - target: ${SPICE_PROXY_PORT}
+        published: ${SPICE_PROXY_PORT}
+        protocol: tcp
+        mode: host
+    networks:
+      - isard_network
+    image: isard/squid:${TAG:-latest}
+    build:
+      context: .
+      dockerfile: dockers/squid/Dockerfile
+    restart: "no"
+
 networks:
   isard_network:
     external: false
     name: isard_network
+#  internet-vms:
+#    driver: macvlan
+#    driver_opts:
+#      parent: viewers
+#    ipam:
+#      config:
+#        - subnet: 172.30.0.0/16
```

Comment:
This comments could be deleted, since they're not used for anything

Reply:
They are needed for vlans